src/main/java/nextstep/subway/common/error/ErrorAdvice.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.subway.common.error;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ErrorAdvice {
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(IllegalArgumentException.class)
+    public void handleIllegalArgumentException() {
+    }
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(IllegalStateException.class)
+    public void handleIllegalStateException() {
+    }
```

Comment:
```suggestion
    }
    
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    @ExceptionHandler(Exception.class)
    public void handleAllException(Exception e) {
        e.printStackTrace();
        log.error("{}", e);
    }
```
요렇게 추가 되면 모든 `Exception`에 대해서 `handle`할 수 있습니다 😄 
이 부분은 참고만 부탁 드리겠습니다 🙇